ArduPlane/Parameters.cpp

```diff
@@ -617,8 +617,8 @@ const AP_Param::Info Plane::var_info[] = {
 
     // @Param: LOG_BITMASK
     // @DisplayName: Log bitmask
-    // @Description: Bitmap of what on-board log types to enable. This value is made up of the sum of each of the log types you want to be saved. It is usually best just to enable all log types by setting this to 65535. The individual bits are ATTITUDE_FAST=1, ATTITUDE_MEDIUM=2, GPS=4, PerformanceMonitoring=8, ControlTuning=16, NavigationTuning=32, Mode=64, IMU=128, Commands=256, Battery=512, Compass=1024, TECS=2048, Camera=4096, RCandServo=8192, Sonar=16384, Arming=32768, FullLogs=65535
-    // @Bitmask: 0:ATTITUDE_FAST,1:ATTITUDE_MED,2:GPS,3:PM,4:CTUN,5:NTUN,6:MODE,7:IMU,8:CMD,9:BATT_MON,10:COMPASS,11:TECS,12:CAMERA,13:RC,14:RANGEFINDER,15:ARM/DISARM,19:IMU_RAW,20:ATTITUDE_FULLRATE,21:VideoStabilization
+    // @Description: Bitmap of what on-board log types to enable. This value is made up of the sum of each of the log types you want to be saved. It is usually best just to enable all basic log types by setting this to 65535.
+    // @Bitmask: 0:Fast Attitude,1:Med Attitude,2:GPS,3:Performance,4:Control Tuning,5:Nav Tuning,7:IMU,8:Mission Cmds,9:Battery Monitor,10:Comapss,11:TECS,12:Camera,13:RC Input-Output,14:Rangefinder,19:Raw IMU,20:Fullrate Attitude,21:Video Stabilization
```

Comment:
If we aren't limited in characters here why not write everything out?

```suggestion
    // @Bitmask: 0:Fast Attitude,1:Medium Attitude,2:GPS,3:Performance,4:Control Tuning,5:Navigation Tuning,7:IMU,8:Mission Commands,9:Battery Monitor,10:Compass,11:TECS,12:Camera,13:RC Input-Output,14:Rangefinder,19:Raw IMU,20:Fullrate Attitude,21:Video Stabilization
```

Reply:
just trying to be economical...long things mean less checkboxes per line and bigger dialogs..wish there was someway to test what things look like in MP...

Reply:
You can build with the Mission Planner specific emitter https://github.com/ArduPilot/ardupilot/blob/master/Tools/autotest/param_metadata/xmlemit_mp.py and replace the file at `C:\Users\username\Documents\Mission Planner`. That should load the new parameters.

There is a button in the crtl+f screen for param gen. That will update the parameters from Michael's master branch code.

Reply:
did not need to...Q_OPTIONS has longggg sentences and they look fine in MP...so changed to full descriptions where it made sense..thanks